packages/framework-core/src/decorators/metadata.ts

```diff
@@ -1,22 +1,14 @@
-import { PropertyMetadata, Class, AnyClass } from '@boostercloud/framework-types'
+import { Class } from '@boostercloud/framework-types'
+import { ClassMetadata, PropertyMetadata } from 'metadata-booster'
 import 'reflect-metadata'
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function getPropertiesMetadata(classType: Class<any>): Array<PropertyMetadata> {
-  const propertyNames = Object.getOwnPropertyNames(new classType())
-  const propertyTypes = Reflect.getMetadata('design:paramtypes', classType)
-  if (propertyNames.length != propertyTypes.length) {
-    // eslint-disable-next-line prettier/prettier
-    throw new Error(`Could not get proper metadata information of ${classType.name}. While inspecting the class, the following properties were found:
-> ${propertyNames.join(', ')}
-But its constructor parameters have the following types:
-> ${propertyTypes.map((type: AnyClass) => type.name).join(', ')}
-They mismatch. Make sure you define all properties as "constructor parameter properties" (see https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties)
-`)
+  const meta: ClassMetadata = Reflect.getMetadata('booster:typeinfo', classType)
+  if (!meta) {
+    console.log(`Could not get proper metadata information of ${classType.name}`)
+    return []
```

Comment:
It was failing before, and that was causing an error when creating the schema. The idea is that instead of falling, it returns an empty value, so the property type will be unknown and set as `JSONObject`, allowing us to have a schema when working with interfaces or custom types.

Reply:
Could this be related to the inconsistencies we saw yesterday in Mayday schema?

Reply:
![image (1)](https://user-images.githubusercontent.com/13258936/115686437-18484500-a351-11eb-9c7d-bd1b32763dc9.png)